adminapi/filters.py

```diff
@@ -158,10 +165,7 @@ def __repr__(self):
         )
 
     def serialize(self):
-        return {type(self).__name__: [
-            v.serialize() if isinstance(v, BaseFilter) else v
```

Comment:
I know it is not possible anymore, however, maybe it would be good to keep that check to have a more defensive implementation?

Reply:
Also (old lines):
* 176
* 218
* 227